danpitrowiski/hw1-99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,59 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+*/
+
+
+let beerTotal = 10;
+let beerMax = beerTotal;
+
+
+function bottlePluralTest(number){
+	if (number === 1){
+		return "bottle";
+	}
+	else {
```

Comment:
_looks around_ 
`else {`? Else what?!

Who left this `else` out here all alone? Hey, Dan! Some monster dropped this`} else {` statement from its `if {` curly bracket!

Reply:
ooo is the `else` suppose to directly follow the `if` is that why it's all separated here?